package.json

```diff
@@ -140,7 +141,8 @@
     "uuid": "^8.3.1",
     "webpack": "^4.35.0",
     "webpack-bundle-analyzer": "^3.7.0",
-    "webpack-cli": "^3.3.5"
+    "webpack-cli": "^3.3.5",
+    "xml2js": "^0.4.23"
```

Comment:
we've got `xml-js` already in the project. Does this one do something different? Or is it better?

Reply:
Oh!  I didn't spot it.
One moment, please.

Reply:
Done.